jsonnet/kube-prometheus/components/alertmanager.libsonnet

```diff
@@ -52,12 +56,14 @@ local defaults = {
       routes: [
         { receiver: 'Watchdog', match: { alertname: 'Watchdog' } },
         { receiver: 'Critical', match: { severity: 'critical' } },
+        { receiver: 'null', match: { alertname: 'InfoInhibitor' } },
```

Comment:
nit: let's move this between `Critical` and  `Watchdog` so all routes related to particular alert names are close together.

Reply:
Done